### PR TITLE
fix for polyvox external CMakeLists in CMake 3.3.2

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -127,13 +127,14 @@ if (APPLE)
   )
   
   if (NOT _OSX_DESIRED_SDK_PATH)
-    message(FATAL_ERROR "Could not find OS X ${OSX_SDK} SDK. Please pass OSX_SDK_PATH to CMake to point us to your SDKs directory.")
+    message(STATUS "Could not find OS X ${OSX_SDK} SDK. Will fall back to default. If you want a specific SDK, please pass OSX_SDK and optionall OSX_SDK_PATH to CMake.")
```
Comment (huffman) [inline CMakeLists.txt]: type, `optional`

**Comment by hifi-gustavo** [comment]
Build console output for commit: 386df86fd16ba2179a30aad1358ef663dcc21375 available at: https://gist.github.com/05ee49062f1c7286bb1b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5915.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5915.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3744632122918f2bf7a7dcde49b192be9b39eae7 available at: https://gist.github.com/05ee49062f1c7286bb1b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5915.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5915.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 29765200614ad7e47fea8016ab77c388d3719fee available at: https://gist.github.com/05ee49062f1c7286bb1b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5915.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5915.dmg

**Pull request merged**